projects/sw-offline-google-analytics/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "sw-offline-google-analytics",
   "version": "1.0.0",
-  "description": "A service worker implementation of the behavior defined in a page's App Cache manifest.",
+  "description": "A library to extend a service worker's behavior, allowing it to retry failed Google Analytics requests.",
```

Comment:
My understanding is the library queues requests while offline and retries them when a network connection is available again. Is that incorrect? If not, it might be worth phrasing this as "A service worker helper library to retry offline Google Analytics requests when a connection is available" (or something better)